lib/eco/inclexer/inclexer.py

```diff
@@ -515,8 +515,10 @@ def relex(self, node):
             try:
                 token = next_token()
                 lookaheads.append(token[2])
-                if token[3][0] is startnode:
-                    past_startnode = True
+                for temp in token[3]:
```

Comment:
It's probably one to two elements most of the time. Worst case scenarios are huge strings.